app/android/app_template/src/org/xwalk/app/template/AppTemplateActivity.java

```diff
@@ -15,12 +15,25 @@
 import org.xwalk.app.XWalkRuntimeActivityBase;
 
 public class AppTemplateActivity extends XWalkRuntimeActivityBase {
+    private final int SYSTEM_UI_OPTIONS = View.SYSTEM_UI_FLAG_LAYOUT_STABLE |
+            View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION |
+            View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN |
+            View.SYSTEM_UI_FLAG_HIDE_NAVIGATION |
+            View.SYSTEM_UI_FLAG_FULLSCREEN |
+            View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY;
+
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
     }
 
     @Override
+    public void onResume() {
+        super.onResume();
+        enterFullscreen();
```

Comment:
@lincsoon You mean home key is hidden without this change? I have tested this function on emulator(android 4.4.2) with skin WVGA800 which has physical keys on the right.
Without calling enterFullscreen() in onResume:
1. Launch app, the virtual home key was always display.
2. If we press the dial key, then back with return key, the virtual home key does not disappear.
3. If we press the home key, then resume app, the virtual home key does not disappear.